src/review/Review.style.tsx

```diff
@@ -1,11 +1,16 @@
 import styled from 'styled-components'
 
 import { space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
 
 export const StyledReview = styled.blockquote`
   padding: ${space.m} 0;
   margin: 0;
 
+  & {
+    ${normalizeHorizontally}
+  }
```

Comment:
Do you need the override? Or can you just do?
```suggestion
  ${normalizeHorizontally}

Reply:
I changed it (I removed the code being overriden too).